LayoutTests/http/tests/site-isolation/iframe-top-level-navigation.html

```diff
@@ -0,0 +1,27 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<script src="/js-test-resources/js-test.js"></script>
+<script>
+
+description("Verifies top-level navigation initiated by iframe can complete");
+jsTestIsAsync = true;
+
+function runTest() 
+{
+    if (window.location.hostname == "127.0.0.1") {
+        if (window.location.hash == "#compete") {
+            testPassed("Main frame is navigated to new url");
+            finishJSTest();
+            return;
+        }
+
+        window.location.hostname = "localhost";
```

Comment:
Why is it necessary to change the mainframe URL here? Can't we just use localhost for the iframe URL to get a remote frame?

Reply:
Nice catch, this is actually unnecessary to trigger the bug. Test updated.